nav2_planner/src/planner_server.cpp

```diff
@@ -270,6 +283,15 @@ PlannerServer::computePlan()
     publishPlan(result->path);
 
     action_server_->succeeded_current(result);
+
+    auto cycle_duration = (now() - start_time).seconds();
+    if (cycle_duration > 1 / expected_planner_frequency_) {
```

Comment:
This is where I was saying that `1/expected` should be what's stored in the class variable so that you're not inverting this every cycle. If 0, then just always don't log.

Reply:
> This is where I was saying that `1/expected` should be what's stored in the class variable so that you're not inverting this every cycle. If 0, then just always don't log.

Ok, i'm changing it. Do you think that `expected_planner_frequency` in this case is right parameter? We are going to comparing durations.